hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/invalidation/BatchInvalidator.java

```diff
@@ -114,10 +112,7 @@ private void pollAndSendInvalidations(String mapName, InvalidationQueue invalida
     }
 
     private List<Invalidation> pollInvalidations(InvalidationQueue invalidationQueue) {
```

Comment:
we can only call this method if queue size greater than batchSize here: https://github.com/ahmetmircik/hazelcast/blob/28c512d56e8e0339de2f84a4339d3ae583e10947/hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/invalidation/BatchInvalidator.java#L91